network-functions/core-network/charts/astri-upf/README.md

```diff
@@ -0,0 +1,64 @@
+
+`SPDX-License-Identifier: Apache-2.0`
+
+`Copyright (c) 2021 Astri Corporation`
+
+**Overview**
+
+ASTRI’s User Plane Function (UPF) can be distributed and deployed close to the users to enable ultra-low latency and time-sensitive usage scenarios in 5G network.
+
+ASTRI’s UPF delivers high network throughput with low packet latency. ASTRI’s UPF can leverage the Dynamic Device Personalization (DDP) capability on Intel XXV710 to achieve fine grade load balancing.
+
+**Prerequisites**
+
+The UPF and UPF agent container require at least 4 CPU core, 8Gi memory for optimized performance. There should be 1024Mb hugepage memory on the host system. It is recommended to have ubuntu 18.04 LTS as the host system.
+

```

Comment:
OpenNESS host os is Cent OS 7.8 for 20.12.02 release and Cent OS 7.9 for 21.03.  The UPF could have container base image as Ubuntu 18.04 LTS but should be deployable on the Edge Node have Cent OS

Reply:
The recommendation for ubuntu as host system is removed at 98619e3.